src/util/statusPoll.ts

```diff
@@ -0,0 +1,109 @@
+export const serviceState = {
```

Comment:
please treat `/lib` as 'util' mentioned in another comment, would be great if we could move `ServiceWatcher` related stuff into `src/lib/ServiceWatcher`